2021.02.1 doesn't install with most recent version of opam

```
===== INSTALL OPAM DEPEXT =====
2.1.0
[ERROR] depext unmet availability conditions, e.g. 'opam-version >= "2.0.0~beta5" & opam-version < "2.1"'
```

--- a/coq_platform_make.sh
+++ b/coq_platform_make.sh
@@ -21,7 +21,7 @@ source shell_scripts/init_cygwin_fixes.sh
 
 source coq_platform_switch_name.sh
 
-export OPAMYES=yes
+export OPAMYES=1
 export OPAMCOLOR=never
 
 ###################### PARAMETERS #####################

--- a/notes/opam-depext.md
+++ b/notes/opam-depext.md
@@ -1,7 +1,22 @@
+# Depext and opam versions
+
+From opam 2.1.0 on, depext is integrated into opam.
+Depext depedencies are treated in the same way as any other dependency.
+
+The Coq Platforms scripts define a shell variable COQ_PLATFORM_OPAM_DEPEXT_COMMAND,
+which is 'depext' for opam < 2.1.0 and 'install' for opam >= 2.1.0, so that one can use
+opam $COQ_PLATFORM_OPAM_DEPEXT_COMMAND to install depext packages.
+
 # general usag examples
+
+## opam < 2.1.0
 
 opam install depext
 opam depext frama-c
+
+## opam >= 2.1.0
+
+opam install frama-c
 
 # depext supported systems:
 

--- a/shell_scripts/install_opam_depext.sh
+++ b/shell_scripts/install_opam_depext.sh
@@ -28,5 +28,5 @@ then
   fi
 
 else
-  COQ_PLATFORM_OPAM_DEPEXT_COMMAND='install'
+  COQ_PLATFORM_OPAM_DEPEXT_COMMAND='install --confirm-level=unsafe-yes'
 fi

--- a/shell_scripts/install_opam_depext.sh
+++ b/shell_scripts/install_opam_depext.sh
@@ -8,15 +8,25 @@
 # Creative Commons CC0 1.0 Universal License
 # See https://creativecommons.org/publicdomain/zero/1.0/legalcode.txt
 
-###################### INSTALL pkg-config #####################
+###################### INSTALL opam depext #####################
 
-echo "===== INSTALL OPAM DEPEXT ====="
+# From opam 2.1.0 on, depext is integrated into opam
 
-opam --version
-
-if [ "$OSTYPE" == cygwin ]
+if [ $(version_to_number $(opam --version)) -lt $(version_to_number 2.1.0) ]
 then
-  $COQ_PLATFORM_TIME opam install depext depext-cygwinports
+  COQ_PLATFORM_OPAM_DEPEXT_COMMAND='depext'
+
+  echo "===== INSTALL OPAM DEPEXT ====="
+
+  opam --version
+
+  if [ "$OSTYPE" == cygwin ]
+  then
+    $COQ_PLATFORM_TIME opam install depext depext-cygwinports
+  else
+    $COQ_PLATFORM_TIME opam install depext
+  fi
+
 else
-  $COQ_PLATFORM_TIME opam install depext
+  COQ_PLATFORM_OPAM_DEPEXT_COMMAND='install'
 fi

--- a/shell_scripts/install_pkg-config.sh
+++ b/shell_scripts/install_pkg-config.sh
@@ -17,5 +17,5 @@
 # install pkg-config if it is not there
 if ! command -v pkg-config &> /dev/null
 then
-  $COQ_PLATFORM_TIME opam depext conf-pkg-config
+  $COQ_PLATFORM_TIME opam ${COQ_PLATFORM_OPAM_DEPEXT_COMMAND} conf-pkg-config
 fi

--- a/shell_scripts/install_prerequisites_coqide.sh
+++ b/shell_scripts/install_prerequisites_coqide.sh
@@ -13,17 +13,17 @@
 # install gtk3 if not there (CoqIDE)
 if ! pkg-config --short-errors --print-errors --atleast-version 3.18 gtk+-3.0
 then
-  $COQ_PLATFORM_TIME opam depext conf-gtk3
+  $COQ_PLATFORM_TIME opam ${COQ_PLATFORM_OPAM_DEPEXT_COMMAND} conf-gtk3
 fi
 
 # install gtksourceview3 if not there (CoqIDE)
 if ! pkg-config --short-errors --print-errors gtksourceview-3.0
 then
-  $COQ_PLATFORM_TIME opam depext conf-gtksourceview3
+  $COQ_PLATFORM_TIME opam ${COQ_PLATFORM_OPAM_DEPEXT_COMMAND} conf-gtksourceview3
 fi
 
 # install adwaita-icon-theme if not there (CoqIDE)
 if ! pkg-config --short-errors --print-errors adwaita-icon-theme
 then
-  $COQ_PLATFORM_TIME opam depext conf-adwaita-icon-theme
+  $COQ_PLATFORM_TIME opam ${COQ_PLATFORM_OPAM_DEPEXT_COMMAND} conf-adwaita-icon-theme
 fi

--- a/shell_scripts/install_prerequisites_gappa.sh
+++ b/shell_scripts/install_prerequisites_gappa.sh
@@ -16,5 +16,5 @@
 
 if [ $(opam var "gappa:installed") == "false" ]
 then
-  opam depext gappa
+  opam ${COQ_PLATFORM_OPAM_DEPEXT_COMMAND} gappa
 fi

